Core/Core/Todos/Todo.swift

```diff
@@ -73,9 +80,14 @@ public final class Todo: NSManagedObject, WriteableModel {
     public var subtitleText: String {
         switch type {
         case .submitting:
-            guard assignment.dueAt != nil else { return assignment.dueText }
-            let format = NSLocalizedString("Due: %@", bundle: .core, comment: "")
-            return String.localizedStringWithFormat(format, assignment.dueText)
+            guard let dueAt = assignment.dueAt else {
+                return NSLocalizedString("No Due Date", bundle: .core, comment: "")
+            }
+            let calendar = Calendar.current
+            let weekday = calendar.shortWeekdaySymbols[calendar.component(.weekday, from: dueAt)]
+            let dueText = Todo.dateFormatter.string(from: dueAt)
+            let format = NSLocalizedString("Due %@, %@", bundle: .core, comment: "")
```

Comment:
Could this be wrong in other locales or timezones, maybe you want to use 
```
DateFormatter.dateFormat(fromTemplate: 
```
with the correct template.

Reply:
Good call, I was able to make it work with a template. I had to separate the date and the time with a localized "at" because the template doesn't support arbitrary strings.

Reply:
Do we really need the "at"?

Reply:
Dave wants it so yeah 🤷‍♂ if we think there will be issues with it I can push back.